### DISCUSSION TITLE
Bugs and feature requests here please!

### DISCUSSION THREAD

**Discussion by HenkMutsaerts** [ExploreASL/ExploreASL#1217] [Q&A]

**Comment by schdempsey** [comment]
BUG: 
preamble: Processing a batch of 10 patients using ExploreASL 1.10.1. All subjects had identical T1 sequences and ASL sequences. 
Issue: sub-008 T1w properly transforms to MNI, but ASL does not. It looks like its in MNI, but translated in the superior direction. I assume it's improper rigid registration of ASL-T1w due to contrast issues. 

Any advice to fix or test T!-ASL registration? Must sub-008 be excluded from population MNI analysis?

**Reply by MDijsselhof** [reply]
We are sorry to hear you have encountered an issue. There could be many reasons why the images have been improperly registered; it would be easiest if you could provide the subject (in BIDS so as to keep everything anonymous) to us to check for pipeline bugs. 
A brief question: Did you analyse the data using Quality = 0? If you could reply to our email (exploreasl.lab@gmail.com) with Henk in CC (h.j.mutsaerts@amsterdamumc.nl) we can keep track of the thread better. Thank you!
